MAINTAINERS.md

```diff
@@ -65,7 +67,15 @@ Respond to enhancement requests, and forum posts. Allocate time to reviewing and
 
 ### Maintain Overall Health of the Repo
 
-Keep the `main` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches. 
+Keep the `main` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches.
+
+### Manage Backlog
+
+Ensure the repo has a prioritized backlog bugs, features, and other GitHub issues. All issues should be tracked in the backlog on a [GitHub project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards). Be clear about the triage status, priority, development status, and target version. See [Alerting Backlog](https://github.com/opensearch-project/alerting/projects/1) for an example.
```

Comment:
Why do issues need to be tracked in a github project board, isn't it enough if they are well triaged?  Making sure this is adopted in a uniform way will be difficult.

Reply:
Tracking it as a list of issues doesn't signal priority in a clear way to the rest of the community. This is important for being transparent as to what are priorities, when they will be delivered, and where there are areas other can jump in and get involved. Individual repos should use a structure that works well for them, but my specific thinking with GitHub project boards is that they are accessible to anyone on the repo with maintainer permissions, and so this seems like a level-playing field approach for any maintainer on a repo. 

Maybe there is a way in the triage process we can be more clear but a few particular questions that are hard to answer with just issues include:
1. What is actively being worked on
2. Where does something sit in terms of priority with the rest of the issues (this could be a data point an individual may use to pick something up to accelerate the work)

Reply:
I see where you're coming from Eli. Priority is usually a PM construct: developers would t-shirt size work items and PMs would prioritize based on value to the user or strategy for the product. You mix the two and you get the actual order in which you want to work on things. 

For an open-source project that is rarely the case because the repo owner doesn't control all the hands and often has conflicting priorities. What is important to you (company X) is not important to me (company Y). In OSS you make no promises to anyone, and divide project features into (1) everything implemented + (2) everything in progress + (3) everything else (backlog). Prioritization is bound to lead to the wrong assumption that something is going to be worked on sooner than later.

Reply:
Could you help me understand the use case for the prioritization information, are there models you'd like to follow?  Maybe  there a great project board + collaboration example?

Reply:
@dblock said it better in his comment :D https://github.com/opensearch-project/.github/pull/72#discussion_r856575693